src/prefect/server/api/server.py

```diff
@@ -602,6 +603,12 @@ async def start_services():
         if prefect.settings.PREFECT_API_EVENTS_STREAM_OUT_ENABLED:
             service_instances.append(stream.Distributor())
 
+        if (
+            prefect.settings.PREFECT_EXPERIMENTAL_ENABLE_CLIENT_SIDE_TASK_ORCHESTRATION
+            and prefect.settings.PREFECT_API_SERVICES_TASK_RUN_RECORDER_ENABLED
```

Comment:
It feels odd to check both settings here. Should they be one setting? Or would it be okay just to check `TASK_RUN_RECORDER`?

Reply:
Good question 🤔 

I put both because `PREFECT_EXPERIMENTAL_ENABLE_CLIENT_SIDE_TASK_ORCHESTRATION` is going to control the "feature" and will be used in other places to control behavior. The `TASK_RUN_RECORDER` setting is just to start the service or not, which seemed consistent with other services.

If this graduates from an experiment we would strip out `PREFECT_EXPERIMENTAL_ENABLE_CLIENT_SIDE_TASK_ORCHESTRATION`.

Reply:
The pattern I've seen is to have one flag governing whether a prefect server starts a particular loop/consumer service (usually `True` and a permanent flag), which is separate from the feature flag governing whether the feature is enabled or not (usually `False` and temporary)